examples/no-undo-manager.html

```diff
@@ -0,0 +1,105 @@
+<!--
```

Comment:
I don't know enough about AMD to comment very intelligently on this file but it looks good to me.